components/omnibar/signed-in/signed-in.hbs

```diff
@@ -0,0 +1,226 @@
+<header style="margin-top: -40px;">
+      <div class="sjc-omnibar">
+        <nav class="navbar">
+          <a class="navbar-brand" href="https://stjude.cloud">
+            <span class="logo"></span>
+          </a>
+          <a class="sjc-title" href="https://stjude.cloud">St. Jude Cloud</a>
+          <span class="portal-title">
```

Comment:
I'd update "portal" to "app" where we see it going forward to keep our language consistent.